.kitchen.yml

```diff
@@ -22,10 +23,11 @@ suites:
   - recipe[marathon::service]
   attributes:
     mesos:
-      version: 0.25.0
+      version: 0.28.1
     marathon:
       flags:
-        master: zk://localhost:2181/mesos
+        master: localhost:5050
```

Comment:
Actually, the test was set up wrongly in the first place. As Mesos is by default set up without ZooKeeper, the ```--master``` flag has to point to the Mesos master, not ZooKeeper (this is why #44 was failing).
Unfortunately, due to https://github.com/mesosphere/marathon/issues/3606 it is still not possible to run Marathon without ZooKeeper, however that is set up by default to be ```zk://localhost:2181/marathon```, so we don't have to configure it.